app/models/setting/ansible.rb

```diff
@@ -80,6 +81,12 @@ def default_settings
                'with Ansible inventory'),
             'Ansible - Ansible Inventory',
             N_('Default Ansible inventory report template')
+          ),
+          set(
+            'ansible_roles_to_ignore',
+            N_('Those Roles will be excluded when importing roles from smart proxy, you can use you can use * wildcard metacharacters'),
```

Comment:
It should probably mention how to specify multiple values. E.g. comma separated values wrapped in quotes or similar. An example wouldn't hurt :-) feel free to use `theforeman.foreman_scap_client` and `theforeman.foreman.*` for that example